uk.co.compucorp.civicrm.tasksassignments/CRM/Tasksassignments/Upgrader.php

```diff
@@ -756,15 +794,66 @@ public function upgrade_1033() {
   }
 
   /**
-   * Renames the case status Ongoing and Resolved to In Progress and Completed,
+   * Renames the case status Ongoing and Resolved to In Progress and Completed.
    *
    * @return bool
    */
   public function upgrade_1034() {
     $this->_relabelCaseStatus('Ongoing', 'In Progress');
     $this->_relabelCaseStatus('Resolved', 'Completed');
+
+    return TRUE;
   }
 
+  /**
+   * Deletes activity types that are not needed by T&W.
+   *
+   * @return bool
+   */
+  public function upgrade_1035() {
+    $this->_uninstallAllComponentActivities([
+      'CiviCampaign',
+      'CiviContribute',
+      'CiviEvent',
+      'CiviMember',
+      'CiviPledge',
+    ]);
+
+    $this->_uninstallActivitiesByLabel([
```

Comment:
`Label` is usually editable, whereas `name` is supposed to be more like an identifier, can we change this to use names instead of labels?